### PR TITLE
[bitnami/spark] Add bitnami/common subchart and update components

### PR DESCRIPTION
**Pull request by andresbono**
Signed-off-by: Andrés Bono <andresbono@vmware.com>

**Description of the change**

- Start using [bitnami/common](https://github.com/bitnami/charts/tree/master/bitnami/common).
- Update components. Major change in the images: Spark is now using Hadoop `3.2.x`.

**Applicable issues**

  - Related to https://github.com/bitnami/bitnami-docker-spark/issues/3.

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/spark/README.md [hunk 1]
```diff
@@ -271,3 +271,13 @@ security.ssl.needClientAuth=true
...
+
+## Upgrading
+
+### To 3.0.0
+
+- This version introduces `bitnami/common`, a [library chart](https://helm.sh/docs/topics/library_charts/#helm) as a dependency. More documentation about this new utility could be found [here](https://github.com/bitnami/charts/tree/master/bitnami/common#bitnami-common-library-chart). Please, make sure that you have updated the chart dependencies before executing any upgrade.
+
+- Spark container images are updated to use Hadoop `3.2.x`: [Notable Changes: 3.0.0-debian-10-r44](https://github.com/bitnami/bitnami-docker-spark#300-debian-10-r44)
+
+> Note: There is no backwards compatibility due to the above mentioned changes. It's necessary to install a new release of the chart, and migrate the existing TSDB data to the new Prometheus instances.
```
Comment (rafariossaa) [inline bitnami/spark/README.md:283]: not very sure if `and migrate the existing TSDB data to the new Prometheus instances.` belongs to here.

**Review by rafariossaa — COMMENTED** [review]
It looks good to me, but check the comment in README.md.

File: bitnami/spark/README.md (hunk 1 above)
Comment (andresbono) [reply bitnami/spark/README.md]: Thanks for the catch!

**Review by rafariossaa — APPROVED** [review]
LGTM

**Pull request merged**